packages/docs/src/routes/docs/concepts/resumable/index.mdx

```diff
@@ -15,8 +15,10 @@ When an SSR/SSG application boots up on a client, it requires that the framework
 1. Listeners - locate event listeners and install them on the DOM nodes to make the application interactive.
 2. Component tree - build up an internal data structure representing the application component tree.
 3. Application state - restore the application state.
+[//]: # (What state are we talking about ? Is there an application-wise state in Qwik ?)
 
 Collectively, this is known as hydration. All current generations of frameworks require this step to make the application interactive.
+[//]: # (What kind of framework are talking about ? It looks like we are talking about SSR/SSG frameworks.)
```

Comment:
yes

I would modify it to say - currently, all SSR frameworks require this step...